package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/answers-headless-react",
-  "version": "1.1.0-beta.13",
+  "version": "1.1.1-alpha.0-122",
```

Comment:
do we need the `.0` part of the tag? I feel like we could just do `1.1.1-alpha-122` or some variant of that